### PR TITLE
localization: move text from wizard to app.json

### PR DESCRIPTION
**Pull request by Tomasvrba**

### CODE REVIEW

File: frontends/web/src/components/devices/bitbox02/bitbox02.tsx [hunk 1]
```diff
@@ -238,89 +239,91 @@ class BitBox02 extends Component<Props, State> {
...
                                     </div>
                                 </Step> : ''}
                             {!unlockOnly && appStatus === 'createWallet' ?
                                 <Step
                                     active={createWalletStatus === 'intro'}
-                                    title="Create Wallet">
+                                    title={t('bitbox02Wizard.create.title')}>
                                     <div className={style.stepContext}>
-                                        <p>Ok, let's create a new wallet! Here are the basics steps you will be taking to setup your BitBox:</p>
+                                        <p>{t('bitbox02Wizard.create.text')}</p>
```
Comment (benma) [inline frontends/web/src/components/devices/bitbox02/bitbox02.tsx]: Split out `Here are the basics steps you will be taking to setup your BitBox` into it's own field, as it's used twice (no need to have it be translated twice).

File: frontends/web/src/components/devices/bitbox02/bitbox02.tsx [hunk 2]
```diff
@@ -330,54 +333,53 @@ class BitBox02 extends Component<Props, State> {
...
                                     </div>
                                 </Step> : ''}
                             {!unlockOnly && appStatus === 'restoreBackup' ?
                                 <Step
                                     active={status !== 'initialized' && restoreBackupStatus === 'intro'}
-                                    title="Restore Backup">
+                                    title={t('backup.restore.confirmTitle')}>
                                     <div className={style.stepContext}>
-                                        <p>Ok, let's restore a backup! Here are the basics steps you will be taking to setup your BitBox:</p>
+                                        <p>{t('bitbox02Wizard.backup.restoreText')}</p>
```
Comment (benma) [inline frontends/web/src/components/devices/bitbox02/bitbox02.tsx]: second instance of the above

File: frontends/web/src/locales/en/app.json [hunk 3]
```diff
@@ -125,6 +125,47 @@
...
+      "paired": "You have confirmed on your device that the code matches. If this is correct, you can continue by clicking the button below.",
+      "title": "Verify your BitBox",
+      "unpaired": "A new BitBox has been detected. Please verify that the following code matches what is shown on your device. If the code matches, touch 'Correct' on your device and then the button below to continue."
+    },
+    "password": {
+      "text": "Now let's set a password for your device. Use the controls on your BitBox to enter and choose a password.",
+      "title": "Set a password for your BitBox"
+    },
+    "success": {
+      "text": "Hooray! You're BitBox is now ready to use, please use the in-app guide on each screen for further information on how to use the app with your BitBox.",
```
Comment (benma) [inline frontends/web/src/locales/en/app.json]: "Your BitBox"

**Comment by Tomasvrba** [comment]
Fixed. Will squash the diff once approved :)

File: frontends/web/src/components/devices/bitbox02/bitbox02.tsx [hunk 4]
```diff
@@ -238,89 +239,91 @@ class BitBox02 extends Component<Props, State> {
...
-                                        <p>Successfully paired your BitBox! Now let's initialize your device. Get started by choosing to create a new wallet, or to restore a wallet from an existing backup. <strong>Please make sure you have a microSD card inserted in your BitBox.</strong></p>
+                                        <SimpleMarkup tagName="p" markup={t('bitbox02Wizard.initialize.text')} />
                                     </div>
                                     <div className={style.buttons}>
                                         <button
                                             className={[style.button, style.primary].join(' ')}
                                             onClick={this.createWalletStep}
                                             disabled={settingPassword || !sdCardInserted}>
-                                            Create Wallet
+                                            {t('bitbox02Wizard.initialize.createButton')}
```
[the remark this answers is not in the document]
Comment (benma) [reply frontends/web/src/components/devices/bitbox02/bitbox02.tsx]: what I meant was literally 'seed.create', not 'bitbox02Wizard.seed.create', as that key already exists and has the same contents and semantics.

**Pull request merged**